pkg/storaged/storage-controls.jsx

```diff
@@ -227,7 +227,7 @@ export const StorageSize = ({ size }) => {
 };
 
 export const StorageMenuItem = ({ onClick, onlyNarrow, danger, excuse, children }) => (
-    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : "") + (danger ? " delete-resource-dangerous" : "")}
+    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : "") + (danger && !excuse ? " delete-resource-dangerous" : "")}
```

Comment:
Oh, this reminds me: we can remove the "onlyNarrow" stuff now, I think.

Reply:
```
pkg/storaged/storage-controls.jsx:export const StorageMenuItem = ({ onClick, onlyNarrow, danger, excuse, children }) => (
pkg/storaged/storage-controls.jsx:    <DropdownItem className={(onlyNarrow ? "show-only-when-narrow" : "") + (danger ? " delete-resource-dangerous" : "")}
pkg/storaged/storage-controls.jsx:export const StorageBarMenu = ({ label, isKebab, onlyNarrow, menuItems }) => {
pkg/storaged/storage-controls.jsx:        <Dropdown className={onlyNarrow ? "show-only-when-narrow" : null}
```

Indeed, nice cleanup for after this.